src/history.rs

```diff
@@ -219,6 +219,22 @@ impl Queue {
     }
 }
 
+#[derive(Debug)]
+pub(crate) enum HistoryError {
```

Comment:
```suggestion
pub enum HistoryError {
```

Sorry, that was a stupid suggestion. That of course will cause the problems we see now on the CI because the other parts here are "more public". As nothing is exported from this module, just keep it like everything else here to reduce the noise.